src/components/toggle/Toggle.tsx

```diff
@@ -82,6 +90,18 @@ const Toggle: GenericComponent<ToggleProps> = ({
             </TextElement>
           )}
           {children}
+          {tooltip && (
+            <TooltippedIcon
+              tooltip={<TextSmall>{tooltip}</TextSmall>}
+              tooltipSize="small"
+              color="teal"
+              tint="darkest"
+              marginLeft={1}
+              {...tooltipProps}
```

Comment:
I think you are already setting all of the tooltip props. Do we need the `tooltipProps`?

Reply:
that is for customisation, in case we need it in codebase

Reply:
I think if we are already setting the icon and its color, we will not be customizing the tooltip itself. Might be worth a check with the design team.

Reply:
There are still lot of props that can be customized:
https://github.com/teamleadercrm/ui/blob/next-release/src/components/tooltip/Tooltip.js#L261

Reply:
I get that and that is what I'm saying. If we are implementing the tooltip directly into the toggle, it is for consistency purposes. Then we should limit the customization because that would be against the consistency.
Now the only difference between this implementation and the original one is the fixed icon. Doesn't look like a big enough reason to change the implementation IMO. And based on this pattern I would argue that in this situation we should hardly ever change the color of the tooltip or show it on click.

Reply:
I will limit customisation props in that case, to keep consistency but allow tweaks that are used in codebase and makes sense.